osgp/protocol-adapter-mqtt/osgp-protocol-adapter-mqtt/src/main/resources/osgp-adapter-protocol-mqtt.properties

```diff
@@ -31,9 +31,17 @@ flyway.init.on.migrate=true
 entitymanager.packages.to.scan=org.opensmartgridplatform.adapter.protocol.mqtt.domain.entities
 
 # MQTT
-mqtt.broker.defaultPort=8883
-mqtt.broker.defaultTopics=+/measurement,+/congestion
-mqtt.broker.defaultQos=AT_LEAST_ONCE
+mqtt.default.host=localhost
+mqtt.default.port=8883
+mqtt.default.qos=AT_LEAST_ONCE
+mqtt.default.topics=+/measurement,+/congestion
+
+mqtt.ssl.enabled=true
+#mqtt.ssl.client.truststore.location=/etc/osp/keys/mqtt/mqtt_client_truststore.jks
```

Comment:
Any reason to keep alternatives commented out. Looks clear enough if this were removed.
If it is intended as documentation, a more explicit comment about allowed values like: "may contain a file name on the classpath, or the full path to the desired file" may be a better fit. A switch from String to Resource would also work, in which case the value already gives a hint that it describes some form of resource (starting with 'classpath:' for instance.)

Reply:
Updated comments.